x-pack/plugins/spaces/server/lib/create_spaces_service.ts

```diff
@@ -4,9 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { DEFAULT_SPACE_ID } from '../../common/constants';
 import { getSpaceIdFromPath } from './spaces_url_parser';
 
 export interface SpacesService {
+  getIsDefaultSpaceId: (req: any) => boolean;
```

Comment:
I'd prefer to name this something like `isInDefaultSpace`, since it's making the determination based off of the user's request, and not a passed `spaceId`.

I don't have a strong preference on what how you define the [tutorial context property](https://github.com/elastic/kibana/pull/22998/files#diff-9afcc0a8a0964cfffbfbf595fec0c17cR13), so if `isDefaultSpaceId` makes more sense there, then feel free!